nsv13/code/modules/jobs/security/weapons.dm

```diff
@@ -0,0 +1,79 @@
+
+/obj/item/gun/ballistic/tazer
+	name = "X24 Tazer"
+	desc = "A stunning weapon developed by Czanek Corp. It can deliver an extremely powerful electric shock via a specialised electrode, though the electrodes must be manually replaced after each shot. <b>It has an effective range of 4 meters</b>"
+	icon = 'nsv13/icons/obj/guns/projectile.dmi'
+	icon_state = "taser"
+	mag_type = /obj/item/ammo_box/magazine/tazer_cartridge
+	can_suppress = FALSE
+	w_class = 2
+	fire_delay = 2 SECONDS
+	can_bayonet = FALSE
+	mag_display = TRUE
+	mag_display_ammo = FALSE
+	bolt_type = BOLT_TYPE_LOCKING
+	slot_flags = ITEM_SLOT_BELT
+	fire_sound = 'sound/weapons/zapbang.ogg'
+	recoil = 2 //BZZZZTTTTTTT
+
+/obj/item/ammo_box/magazine/tazer_cartridge
+	name = "X24 Tazer cartridge"
+	desc = "A cartridge which can hold a taser electrode"
```

Comment:
Any chance of picking which spelling to use?

Reply:
nope lol

Reply:
Yes. taser.